src/Roact.lua

```diff
@@ -9,7 +9,8 @@ if not roactModule then
 		local TS = require(game:GetService("ReplicatedStorage"):WaitForChild("rbxts_include"):WaitForChild("RuntimeLib"))
 		Roact = TS.import(script, TS.getModule(script, "roact").src)
 	else
-		error("Roact Router failed to find Roact. Did you make sure Roact is in the same folder?")
+		local TS = require(container.Parent.Parent.RuntimeLib)
+		Roact = TS.import(script, TS.getModule(script, "roact").src)
```

Comment:
This seems to be a clone of the line in the if statement prior to this. Why not just use a ternary statement when importing `TS`?